data/assets/scene/solarsystem/heliosphere/2012/sun_earth_2012_fieldlines_batsrus.asset

```diff
@@ -0,0 +1,214 @@
+    ColorTablePaths = {
+      batsrusTemperatureColorTable,
+      batsrusDensityColorTable,
+      batsrusCurrentColorTable,
+      batsrusVelocityColorTable,
+      batsrusTopologyColorTable,
+    },
+   ColorTableRanges = {
+      { 0, 100000000 },
+      { 0, 60 },
+      { -0.015, 0.015 },
+      { 150, 900 },
+      { 0, 3 },
+    },
+    --[[ ExtraMinMaxLimits = {
+      { 0, 1000000000 },
+      { -1, 150 },
+      { -0.2, 0.2 },
+      { 0, 6000 },
+      { 0, 3 },
+    },
+    RadialDomainLimits = { 0, 350 },
+    CartesianDomainLimits = { { -400, 60 }, { -80, 80 }, { -100, 100 } }, ]]
+    LoadAtRuntime = true
+  },
+  GUI = {
+    Name = "Fieldlines BATSRUS Asher Static",
+    Path = "/Solar System/Heliosphere"
+  }
+}
+
+asset.onInitialize(function ()
+  openspace.addSceneGraphNode(BatsrusJ12OpenClosed)
+  openspace.addSceneGraphNode(BatsrusJ12FlowLines)
+  openspace.addSceneGraphNode(BatsrusAsherStaticSeedsFlowLines)
+
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_OpenClosed.Renderable.Color.colorMethod", 1) -- Color by Quantity
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_OpenClosed.Renderable.Color.colorQuantity", 4) -- Topology
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_OpenClosed.Renderable.Domain.limitsY", { -30.61, 30.61 })
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_OpenClosed.Renderable.Domain.limitsY", { -30.61, 30.61 })
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_OpenClosed.Renderable.maskingEnabled", true)
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_OpenClosed.Renderable.Masking.maskingQuantity", 4) -- Topology
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_OpenClosed.Renderable.Masking.maskingMinLimit", "2.5")
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_OpenClosed.Renderable.Masking.maskingMaxLimit", "3.0")
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_FlowLines.Renderable.Color.colorMethod", 1) -- Color by Quantity
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_FlowLines.Renderable.Color.colorQuantity", 3) -- Velocity
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_FlowLines.Renderable.Color.colorQuantityMin", "75")
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_FlowLines.Renderable.Color.colorQuantityMax", "350")
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_FlowLines.Renderable.Color.uniform", { 1, 1, 1, 0.12 })
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_FlowLines.Renderable.Flow.particleSize", 2)
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_FlowLines.Renderable.Flow.particleSize", 2)
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_FlowLines.Renderable.Flow.particleSpacing", 60)
+  -- openspace.setPropertyValueSingle("Scene.FL_BATSRUS_J12_FlowLines.Renderable.Flow.speed", 20)
+end)
+
+asset.onDeinitialize(function ()
+  openspace.removeSceneGraphNode(BatsrusJ12OpenClosed)
```

Comment:
It's usually a good idea to have the scene graph nodes in reverse order here from the onInitialize.  That way we won't get any weird behavior if nodes depend on each other. So:
```lua
openspace.removeSceneGraphNode(BatsrusAsherStaticSeedsFlowLines)
openspace.removeSceneGraphNode(BatsrusJ12FlowLines)
openspace.removeSceneGraphNode(BatsrusJ12OpenClosed)
```